tests/visual/dropdowns.html

```diff
@@ -144,17 +144,18 @@
             <span class="col-2"></span>
                 <div class="k-popup k-list-container k-reset" >
                     <kendo-list>
-                        <div class="k-outer-group-header"> <strong>GROUP 1</strong> </div>
+                        <div class="k-outer-group-header k-first"> <strong>GROUP 1</strong> </div>
```

Comment:
Can you name it so that it is clear to indicate first item and that should have box-shadow? For example: `<strong> First group item with box-shadow</strong>`?